tools/SetupFlow/DevHome.SetupFlow/Models/RepositoryProvider.cs

```diff
@@ -50,6 +51,11 @@ internal sealed class RepositoryProvider
     /// </summary>
     private IRepositoryProvider _repositoryProvider;
 
+    /// <summary>
+    /// Used to enable searching for repos with a query.
+    /// </summary>
+    private IRepositoryProvider2 _repositoryProvider2;

```

Comment:
I wonder if this is the right way to do this. We're creating copies of repositoryProvider for each interface it implements.

Reply:
Removed it.